addons/crm/wizard/crm_lead_lost.py

```diff
@@ -9,20 +9,25 @@ class CrmLeadLost(models.TransientModel):
     _name = 'crm.lead.lost'
     _description = 'Get Lost Reason'
 
+    lead_ids = fields.Many2many('crm.lead')
```

Comment:
This is m2m instead of o2m because I didn't want to add a field for a transient model...